kontrol-service/api/server.go

```diff
@@ -67,27 +67,27 @@ func (sv *Server) GetTenantUuidFlows(_ context.Context, request api.GetTenantUui
 
 	finalTopology := flow.MergeClusterTopologies(*clusterTopology, lo.Values(allFlows))
 	flowHostMapping := finalTopology.GetFlowHostMapping()
-
 	resp := lo.MapToSlice(flowHostMapping, func(flowId string, flowUrls []string) apitypes.Flow {
-		isBaselineFlow := flowId == prodFlowId
+		// the baseline flow ID uses the base cluster topology namespace name
+		isBaselineFlow := flowId == clusterTopology.Namespace
 		return apitypes.Flow{FlowId: flowId, FlowUrls: flowUrls, IsBaseline: &isBaselineFlow}
 	})
 	return api.GetTenantUuidFlows200JSONResponse(resp), nil
 }
 
 func (sv *Server) PostTenantUuidDeploy(_ context.Context, request api.PostTenantUuidDeployRequestObject) (api.PostTenantUuidDeployResponseObject, error) {
-	logrus.Infof("deploying prod cluster for tenant '%s'", request.Uuid)
+	logrus.Infof("deploying prod/baseline cluster for tenant '%s'", request.Uuid)
```

Comment:
I thought we did not want the term `prod` to appear anywhere because it could be confusing to the user.

Reply:
@laurentluce you are right, I'll remove it